custom_components/tuya_local/devices/aygrochy_inline_wifi_air_vent.yaml

```diff
@@ -0,0 +1,314 @@
+name: Aygrochy fan
+primary_entity:
+  entity: climate
+  dps:
+    - id: 2
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: "ON"
+          value: "ON"
+        - dps_val: "OFF"
+          value: "OFF"
+        - dps_val: "TIMER"
+          value: "TIMER"
+        - dps_val: "AUTO"
+          value: "AUTO"
+        - dps_val: "ALARM"
+          value: "ALARM"
+    - id: 8
+      readonly: true
+      name: current_humidity
+      type: integer
+      range:
+        min: 0
+        max: 255
+      unit: "%"
+    - id: 9
+      readonly: true
+      name: current_temperature
+      type: integer
+      range:
+        min: -40
+        max: 255
+      unit: "F"
+    - id: 106
+      name: target_temp_high
+      type: integer
+      range:
+        min: 32
+        max: 194
+      unit: "F"
+    - id: 107
+      name: humidity
+      type: integer
+      range:
+        min: 0
+        max: 100
+      unit: "%"
+
+secondary_entities:
+  - entity: fan
+    name: fan
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+      - id: 102
+        name: speed
+        type: integer
+        range:
+          min: 0
+          max: 10
+  - entity: light
+    name: Backlight brightness
+    dps:
+      - id: 101
+        name: brightness
+        type: integer
+        range:
+          min: 0
+          max: 4
+  - entity: sensor
+    name: Current speed
+    dps:
+      - id: 103
+        name: sensor
+        readonly: true
+        type: number
+  - entity: number
+    name: Countdown timer
+    dps:
+      - id: 104
+        name: value
+        type: integer
+        unit: s
+        range:
+          min: 0
+          max: 36000
+  - entity: sensor
+    name: Alarm triggered
```

Comment:
Should maybe be an event. But the values are needed (or at least the value when an alarm is not triggered, so it can be filtered out).